tdp/cli/commands/browse.py

```diff
@@ -118,7 +118,7 @@ def _print_formatted_deployment(deployment_log: DeploymentLog) -> None:
     )
     if deployment_log.component_version:
         click.echo(
-            "\Component verion logs:\n"
+            "\nComponent verion logs:\n"
```

Comment:
I don't want to abuse, but could you also correct the word "verion" plz ? ^^'

Reply:
True haha

Done.